index.html

```diff
@@ -123,212 +123,9 @@ <h2 class="hone">Facebook 360</h2>
-      <div class="table">
-        <h2>Tonic</h2>
-        <ul>
-           <li><h3>CANOPY</h3></li>
-           <li class="list"><img  src="./assets/Images/ball.png" alt="pic"></li>
-            <li>Back End Dev</li>
-            <li class="list"><img  src="./assets/Images/ball.png" alt="pic">
-            <li>2015</li>
-        </ul>
-        <p class="paraone">A daily selection of privately
-            personalized reads; no accounts or sign-ups required.</p>
-            <ul>
-                <li class="language">Css</li>
-                <li class="language">Html</li>
-                <li class="language">Javascript</li>
-            </ul>
-            <div class="button"><a href="#" class="color">See Project</a></div>
-       </div>
-    </div> 
-</section>
-<section class="cards">
-    <div class="cardernan">
-      <img src="./assets/Images/imagetwo.png" alt="Calculator">
-      <div class="table2">
-        <h2 class="hone">Multi-Post
-            Stories</h2>
-            <ul>
-                <li><h3>CANOPY</h3></li>
-                <li class="list"><img  src="./assets/Images/ball.png" alt="pic">
-                 <li>Back End Dev</li>
-                 <li class="list"><img  src="./assets/Images/ball.png" alt="pic">
-                 <li>2015</li>
-             </ul>
-        <p class="paraone">Experimental content creation feature 
-            that allows users to<br> add to an existing story over the
-             course of a day without <br>spamming their friends.</p>
-            <ul>
-                <li class="language">Css</li>
-                <li class="language">Html</li>
-                <li class="language">Javascript</li>
-            </ul>
-            <div class="button"><a href="#" class="color">See Project</a></div>
-       </div>
-    </div>
-</section>
 <section class="cards">
     <div class="cardern">
-      <img src="./assets/Images/numberfour.png" alt="pro">
-      <div class="table">
-        <h2 class="hone">Facebook 360</h2>
-        <ul>
-            <li><h3>CANOPY</h3></li>
-            <li class="list"><img  src="./assets/Images/ball.png" alt="pic">
-             <li>Back End Dev</li>
-             <li class="list"><img  src="./assets/Images/ball.png" alt="pic">
-             <li>2015</li>
-         </ul>
-        <p class="paraone">Exploring the future of media in Facebook's first Virtual Reality app;
-             a place to discover and enjoy 360 photos and videos on Gear VR..</p>
-            <ul>
-                <li class="language">Css</li>
-                <li class="language">Html</li>
-                <li class="language">Javascript</li>
-            </ul>
-            <div class="button"><a href="#" class="color">See Project</a></div>
-       </div>
-    </div>
-</section>
-<section class="cards">
-    <div class="cardernan">
       <img src="./assets/Images/numbertwo.png" alt="image">
       <div class="table">
         <h2 class="hone">Uber Navigation</h2>
```

Comment:
Kindly as requested before and as this is required  make all this work section cards get filled dynamically from javascript code the same way you did to fill the popup window and from the same array